### PR TITLE
Reader: enable process with a generator

### PR DESCRIPTION
**Pull request by cchampet**

### CODE REVIEW

File: src/AvTranscoder/reader/IReader.cpp [hunk 1]
```diff
@@ -70,10 +73,20 @@ Frame* IReader::readFrameAt(const size_t frame)
...
+        if(!_continueWithGenerator)
+        {
+            _dstFrame->clear();
+            return _dstFrame;
+        }
+        // or generate data (ie silence or black)
+        else
+        {
+           _generator->decodeNextFrame(*_srcFrame);
+        }
```
Comment (valnoel) [inline src/AvTranscoder/reader/IReader.cpp]: You could have done : 
```c++
// generate data (ie silence or black)
if(_continueWithGenerator)
{
    _generator->decodeNextFrame(*_srcFrame);
}
// or return an empty frame
else
{
    _dstFrame->clear();
    return _dstFrame;
}
```
or
```c++
// return an empty frame
if(!_continueWithGenerator)
{
    _dstFrame->clear();
    return _dstFrame;
}
// or generate data (ie silence or black)
_generator->decodeNextFrame(*_srcFrame);
```

**Comment by cchampet** [comment]
Need to up to v0.9.2

File: src/AvTranscoder/reader/IReader.cpp (hunk 1 above)
Comment (cchampet) [inline src/AvTranscoder/reader/IReader.cpp]: Done here 7dd618a4a23d9fa05157472b9a0cc7856af7aec5

**Comment by cchampet** [comment]
See PR #239

**Pull request closed without merging**